src/model/network-simple-multicast.hpp

```diff
@@ -154,6 +159,9 @@ class SimpleMulticastNetwork : public Network
 
   // Floorplanner interface.
   void SetTileWidth(double width_um);
+
+  // Parse ERT to get multi-casting energy
+  double getMulticastEnergy(std::uint64_t multicast_factor);
```

Comment:
Same as previous comment.

Reply:
I have updated the function names, thanks!